### PR TITLE
Free-busy spaces

### PR DESCRIPTION
**Pull request by christophermark**
Added some endpoints for fetching free-busy objects for spaces.

### CODE REVIEW

File: src/main/java/com/robinpowered/sdk/model/Busy.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
+package com.robinpowered.sdk.model;
+
+import org.joda.time.DateTime;
+
+import java.util.List;
+
+public class Busy {
```
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/Busy.java:14]: Missing class docblock, as well as doc blocks for all public members. Look at other models for some pasta

File: src/main/java/com/robinpowered/sdk/model/FreeBusySpace.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        this.space = space;
+        this.busy = busy;
+    }
+
+    @Override
+    public String getMimeType() {
+        return MIME_TYPE;
+    }
+
+    public boolean isHasPresence() {
```
Comment (brianmuse) [inline src/main/java/com/robinpowered/sdk/model/FreeBusySpace.java]: no need for `is`

**Comment by brianmuse** [comment]
Great stuff, just needs doc blocks on several things. Any class, interface, interface method, or public member.

File: src/main/java/com/robinpowered/sdk/model/FreeBusySpace.java (hunk 2 above)
Comment (christophermark) [inline src/main/java/com/robinpowered/sdk/model/FreeBusySpace.java]: My bad. This was autogenerated.

**Comment by christophermark** [comment]
All right, docs added.

**Comment by brianmuse** [comment]
Great stuff here! :+1:

**Pull request merged**